packages/nextjs/src/config/webpack.ts

```diff
@@ -195,22 +159,20 @@ function addFileToExistingEntryPoint(
   const currentEntryPoint = entryProperty[entryPointName];
   let newEntryPoint: EntryPointValue;
 
-  // We inject the user's client config file after the existing code so that the config file has access to
-  // `publicRuntimeConfig`. See https://github.com/getsentry/sentry-javascript/issues/3485
   if (typeof currentEntryPoint === 'string') {
-    newEntryPoint = [currentEntryPoint, filepath];
+    newEntryPoint = [filepath, currentEntryPoint];
```

Comment:
> We inject the user's client config file after the existing code [...]

This comment was removed. Is there any way to validate we won't be running in the same [issue](https://github.com/getsentry/sentry-javascript/issues/3485)? Also checked the tests, and while I may have not seen the one, there aren't any tests for that issue.

Reply:
I'll explain the details in the PR description, but FWIW I tested it manually and users are indeed still able to access `publicRuntimeConfig`, both from within API routes and regular pages (both server and browser). As for automated tests, they would need to be integration tests. I'll look into how complicated that would be to set up with Kamil's framework.